sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -1180,8 +1236,27 @@ export interface PathSetPermissionsOptions extends CommonOptions {
     owner?: string;
 }
 
-// Warning: (ae-forgotten-export) The symbol "PathUpdateHeaders" needs to be exported by the entry point index.d.ts
-//
+// @public
+export interface PathUpdateHeaders {
+    acceptRanges?: string;
+    cacheControl?: string;
+    contentDisposition?: string;
+    contentEncoding?: string;
+    contentLanguage?: string;
+    contentLength?: number;
+    contentMD5?: string;
+    contentRange?: string;
+    contentType?: string;
+    date?: Date;
```

Comment:
Are all of these actually optional? E.g. feels like `date` would always be present?

Reply:
Same question for all the *Header types.

Reply:
`date` is required when using shared key authorization. So will have to leave it as optional to fit all scenarios.
```
x-ms-date |   | string | Specifies the Coordinated Universal Time (UTC) for the request. This is required when using shared key authorization.
```
https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update#request-headers

Will do a thorough check for these headers.

Reply:
Also note that we set the the "x-ms-date" in the `StorageSharedKeyCredentialPolicy.signRequest()`.

```
  protected signRequest(request: WebResource): WebResource {
    request.headers.set(HeaderConstants.X_MS_DATE, new Date().toUTCString());
  ...
  }
```

Reply:
I checked `PathHttpHeaders` and `PathCreateHttpHeaders`. I think we are good with all these optional.

Reply:
Great thanks!